README.md

```diff
@@ -112,3 +125,7 @@ over time, is inspired by a wide body of research in applied category theory and
 beyond. Incomplete bibliographies are in the [dev
 docs](https://next.catcolab.org/dev/bib-0001.xml) and the [core
 docs](https://next.catcolab.org/dev/rust/catlog/refs).
+
+
+## TODO: move to dev docs
```

Comment:
I think it's fine to leave have the above in the README, but this TODO should be removed from the README.